### ISSUE TITLE
calculate_hand does not execute immediately

### ISSUE THREAD

**Issue by cirrusasf** [ASFHyP3/asf-tools#103]
**Describe the bug**
When I fire up calculate_hand without any argument, it takes a while to show up help information:
usage: calculate_hand [-h] [-v] out_raster vector_file
calculate_hand: error: the following arguments are required: out_raster, vector_file

**Comment by jhkennedy** [comment]
Yes, this is a problem with the asf-tools container right now (I also meant to open an issue). It's order seconds but 10s of them.

I tried removing this to see if it helped and it does not:
https://github.com/ASFHyP3/asf-tools/blob/develop/Dockerfile#L15

**Comment by jhkennedy** [comment]
Oh, interesting. I was only seeing it in the docker container the first time, and not locally for me. :thinking:

**Comment by asjohnston-asf** [comment]
See if bringing your local conda environment up to date introduces the issue? Could have been introduced by a dependency version bump.

**Comment by jhkennedy** [comment]
Some timing notes:

Both of these commands:
```
calculate_hand --help
water_map --help
```
takes ~1.65 seconds in my environment

It's something about the imports in `water_map.py` and/or `hand/caculate.py`  I think as it takes 0.04s to run:
```
python -c "import asf_tools; print(asf_tools.__version__)"
```

but the same ~1.65 seconds to run:
```
python -c "import asf_tools.water_map; print(asf_tools.__version__)"
```

**Comment by jhkennedy** [comment]
On the docker side, the times are even worse the first time they are run ( run each right after a fresh `docker run`)

```
calculate_hand --help
water_map --help
```
takes ~31.5 seconds

```
python -c "import asf_tools; print(asf_tools.__version__)"
```
takes ~0.07 seconds

```
python -c "import asf_tools.water_map; print(asf_tools.__version__)"
```
takes ~31.5 seconds

--- 

The second time I run the above commands in a running container, the ~31.5 seconds times drop to ~3 seconds

**Comment by jhkennedy** [comment]
And completely wiping my local conda environment and recreating sees the same thing as the docker container 31s the first time, 3 seconds the second.

**Comment by cirrusasf** [comment]
I just tested it. it takes ~3 secs.
